cmd/boulder-sa/main.go

```diff
@@ -20,6 +20,9 @@ type config struct {
 		cmd.DBConfig
 
 		Features map[string]bool
+
+		// Max simultaneous SQL queries caused by a single RPC.
+		ParallelismPerRPC int
```

Comment:
Could you make this a `uint`? (_Is this too nit picky a nit?_)

Reply:
In general I don't like to use `uint` as an indicator of "this is never negative." While it doesn't apply in this case, the risk in general is that people use that to skip bounds checking and assume the value is always good. But in a lot of cases, 65535 is just as bad as -1.

In this particular case, since the value is coming from a config file and isn't incremented or decremented, not a big deal. And the JSON parser actually does reject negative values when parsing into a `uint`, instead of wrapping them around. However, when I made this change, I realized it would also propagate into the constructor for NewSQLStorageAuthority, etc. So I decided to go with a simple bounds check.

Reply:
WFM